models/issue_comment.go

```diff
@@ -86,6 +86,8 @@ const (
 	CommentTypeChangeTargetBranch
 	// Delete time manual for time tracking
 	CommentTypeDeleteTimeManual
+	// add or remove Request from one
+	CommentTypeReviewRequest
```

Comment:
I don't think it's necessary to distinguish between them because they have the same information (doer and  freviewer )， and it will make code simple if make them as one type